### DISCUSSION TITLE
Emulating the "Copy all as cURL" feature?

### DISCUSSION THREAD

**Discussion by gitcnd** [Tampermonkey/tampermonkey#1675] [Q&A]
Is anyone aware of a sample script which can output loaded page resources in the "copy all as cURL" format (you know - with all the headers and cookies and so on all on one line) ?

If not - does anyone know if sufficient tapermonkey features exist to make such an output possible?  (e.g. access to cookies and other headers that got sent, callbacks or triggers on network events, etc).

Alternatively - can Tampermonkey access the chrome "inspect" subsystem?, and maybe automate doing the "copy all as cURL" actions on all the network items that show up there?

**Comment by rruofan** [comment]
您好，来信已收到~我会尽快处理

**Comment by derjanb** [comment]
> (you know - with all the headers and cookies and so on all on one line)

This is out of the scope and access options of userscripts and therefore needs a new browser extension.
